tests/issues/issue-1733.php

```diff
@@ -302,4 +302,90 @@
+        'properties' => [
+            'Comment' => 'Samsung Browser 5.4',
+            'Browser' => 'Samsung Browser',
+            'Browser_Type' => 'Browser',
+            'Browser_Bits' => '32',
+            'Browser_Maker' => 'Samsung',
+            'Browser_Modus' => 'unknown',
+            'Version' => '5.4',
+            'Platform' => 'Android',
+            'Platform_Version' => '6.0',
+            'Platform_Description' => 'Android OS',
+            'Platform_Bits' => '32',
+            'Platform_Maker' => 'Google Inc',
+            'Alpha' => false,
+            'Beta' => false,
+            'Frames' => true,
+            'IFrames' => true,
+            'Tables' => true,
+            'Cookies' => true,
+            'JavaScript' => true,
+            'VBScript' => false,
+            'JavaApplets' => false,
+            'isSyndicationReader' => false,
+            'isFake' => false,
+            'isAnonymized' => false,
+            'isModified' => false,
+            'CssVersion' => '3',
+            'Device_Name' => 'Galaxy S7 Edge',
+            'Device_Maker' => 'Samsung',
+            'Device_Type' => 'Mobile Phone',
+            'Device_Pointing_Method' => 'touchscreen',
+            'Device_Code_Name' => 'SM-G935F',
+            'Device_Brand_Name' => 'Samsung',
+            'RenderingEngine_Name' => 'Blink',
+            'RenderingEngine_Version' => 'unknown',
+            'RenderingEngine_Maker' => 'Google Inc',
+        ],
+        'lite' => false,
+        'standard' => true,
+        'full' => true,
+    ],
+    'issue-1733-I' => [
+        'ua' => 'Mozilla/5.0 (Linux; Android 6.0.1; SAMSUNG SM-J510FN Build/MMB29M) AppleWebKit/537.36 (KHTML, like Gecko) SamsungBrowser/6.2 Chrome/56.0.2924.87 Mobile Safari/537.36',
+        'properties' => [
+            'Comment' => 'Samsung Browser 6.2',
+            'Browser' => 'Samsung Browser',
+            'Browser_Type' => 'Browser',
+            'Browser_Bits' => '32',
+            'Browser_Maker' => 'Samsung',
+            'Browser_Modus' => 'unknown',
+            'Version' => '6.2',
+            'Platform' => 'Android',
+            'Platform_Version' => '6.0',
+            'Platform_Description' => 'Android OS',
+            'Platform_Bits' => '32',
+            'Platform_Maker' => 'Google Inc',
+            'Alpha' => false,
+            'Beta' => false,
+            'Frames' => true,
+            'IFrames' => true,
+            'Tables' => true,
+            'Cookies' => true,
+            'JavaScript' => true,
+            'VBScript' => false,
+            'JavaApplets' => false,
+            'isSyndicationReader' => false,
+            'isFake' => false,
+            'isAnonymized' => false,
+            'isModified' => false,
+            'CssVersion' => '3',
+            'Device_Name' => 'Galaxy S7 Edge',
```

Comment:
Please fix the tests

Reply:
oops. Sorry for that. Will have a look how this happened